Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -27,6 +27,10 @@ namespace testutil {
 
 // Below are convenience methods for creating instances for tests.
 
+inline model::DocumentKey DocKey(absl::string_view path) {
```

Comment:
This function is just called key on Android and JS; let's call it that here too.

Reply:
done